td-logger.gemspec

```diff
@@ -32,7 +32,7 @@ EOF
   gem.require_paths = ['lib']
 
   gem.add_dependency "msgpack", ">= 0.5.6", "< 2.0"
-  gem.add_dependency "td-client", ">= 0.8.66", "< 2.0"
+  gem.add_dependency "td-client", ">= 0.8.66", "< 3.0"
```

Comment:
Since 0.8.66 is too old, it may be able to restrict version to 1.0.8...?

Reply:
Sounds good (y)

Reply:
Updated:

```diff
- gem.add_dependency "td-client", ">= 0.8.66", "< 3.0"
+ gem.add_dependency "td-client", ">= 1.0.8", "< 3.0"
```